Manual.md

```diff
@@ -1272,6 +1281,124 @@ cache during the install/removal of the package
 - `font_dirs`: which should be set to the directory where the package
 installs its fonts
 
+<a id="xbps_triggers"></a>
+### XBPS Triggers
+
+XBPS triggers is a collection of snippets of code, provided by the `xbps-triggers`
+package, that are added to the INSTALL/REMOVE
+scripts of packages either manually by setting the `triggers` variable in the template,
+or automatically, when specific conditions are met.
+
+The following is a list for each trigger available, its current status, what it does
+and what conditions need to be met so that it is included automatically on a package.
+
+It is not a complete overview of the package it is reccomended to read the variables
```

Comment:
"This is not" and so on sounds a little better IMO. Also, I'd place a full stop after "of the package"